examples/vmck.nomad

```diff
@@ -0,0 +1,148 @@
+# TODO: remove hardcoded values
+#       add configurationa like in https://github.com/liquidinvestigations/node/blob/master/templates/drone.nomad
+job "vmck" {
+  datacenters = ["dc1"]
+  type = "service"
+
+  group "imghost" {
+    task "nginx" {
+      driver = "docker"
+      config {
+        image = "nginx:mainline"
+        volumes = [
+          "/usr/share/vmck-images:/usr/share/nginx/html", # TODO: add setting for host volume
+          "local/nginx.conf:/etc/nginx/nginx.conf",
+        ]
+        port_map {
+          http = 80
+        }
+        labels {
+          liquid_task = "vmck-imghost-nginx"
```

Comment:
You can remove the label, it made sense in the context of the liquid node tooling, where it's common to shell into docker containers and run commands. For this repo we can skip it.